SQL/Archive/20.0/2018-04-05_RemovePendingStagingFromFiles.sql

```diff
@@ -0,0 +1,5 @@
+-- Remove the PendingStaging field from the files table as discussed during
```

Comment:
this should go to new patches directory

Reply:
@zaliqarosli Nice catch! I moved the patch to the `Cleanup_patches` directory :). Please re-review.